anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/source/data/DataSource.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.anomdetect.source.data;
+
+import java.util.List;
+
+/**
+ * Data source interface
+ * <p>
+ * Fetches metric data from the source for the specified metric.
+ * Currently it supports only graphite as data source.
```

Comment:
This comment line is not required.  If/When it becomes invalidated (i.e. a new DataSource implementation is added) this comment is unlikely to be updated and will become confusing. Please remove.